app/questionnaire/questionnaire_schema.py

```diff
@@ -116,6 +121,49 @@ def placeholder_transform_section_dependencies_by_block(
     def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    @cached_property
+    def min_and_max_map(self) -> Any:
+        return make_immutable(self._min_and_max_map)
+
+    def _create_min_max_map(
+        self,
+        min_max: Literal["minimum", "maximum"],
+        answer_id: str,
+        answers: Iterable[ImmutableDict],
+        default_min_max: str,
+    ) -> None:
+        longest_value_length = 0
+        for answer in answers:
+            value = answer.get(min_max, {}).get("value")
+
+            if isinstance(value, float):
```

Comment:
My comment wasn't that this is a float, it was that could it also be a float. Looking at validator, it is int or string (string is invalid), I would change to support int and float, as we can make validator allow float, which is edge case but still valid.

Right now setting to a string breaks validator, which is a gaps in validator that can be fixed separately.